lib/kino/smart_cell.ex

```diff
@@ -24,7 +24,7 @@ defmodule Kino.SmartCell do
 
   Here is an outline of a custom module
 
-      defmodule Kino.SmartCell.Custom do
+      defmodule KinoDocs.SmartCell.Custom do
```

Comment:
This would be closest to how we've been calling them in `kino_*` projects :D

```suggestion
      defmodule KinoDocs.CustomCell do
```